ui/src/App.scss

```diff
@@ -1,16 +1,27 @@
 .app {
-  width: 100%;
-  height: 100%;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
   background-color: #eecbcb;
   overflow-y: auto;
   overflow-x: hidden;
-  position: fixed;
+  &__content {
+    margin-bottom: 5rem;
+  }
+}
+
+body, #root {
```

Comment:
Why was this added, i removed it and noticed 0 difference

Reply:
We dont need it any more i have removed it